data/Store.js

```diff
@@ -297,6 +297,8 @@ export class Store {
      * @return {Record}
      */
     getById(id, fromFiltered = false) {
+        if (id == this.summaryRecord?.id) return this.summaryRecord;
```

Comment:
Thanks for mixing this in.  Here's one case where I would recommend `===` for some extra safety.  Not sure that we would h😉